app/bibleview-js/src/components/TextEditor.vue

```diff
@@ -103,18 +103,50 @@ export default {
       title: 'Insert bible reference',
       result: async () => {
         const originalRange = document.getSelection().getRangeAt(0);
+        //Get the active languages and create a bible reference parser for each language
+        const languages = android.getActiveLanguages()
```

Comment:
We should cache this. TextEditors are used extensively and querying this from jvm side every time does not make sense.